spec/acceptance/throttling_spec.rb

```diff
@@ -34,6 +34,27 @@
     end
   end
 
+  it "returns correct Retry-After header if enabled" do
+    begin
+      prev = Rack::Attack.throttled_response_retry_after_header
+      Rack::Attack.throttled_response_retry_after_header = true
+
+      Rack::Attack.throttle("by ip", limit: 1, period: 60) do |request|
+        request.ip
+      end
+
+      get "/", {}, "REMOTE_ADDR" => "1.2.3.4"
+      assert_equal 200, last_response.status
+
+      Timecop.travel(25) do
+        get "/", {}, "REMOTE_ADDR" => "1.2.3.4"
+        assert last_response.headers["Retry-After"].to_i <= 60 - 25
+      end
+    ensure
+      Rack::Attack.throttled_response_retry_after_header = prev
```

Comment:
I think we would not need this if this new config setting is cleared as part of `clear_configuration`.